lib/tarantool_server.py

```diff
@@ -761,7 +761,7 @@ def deploy(self, silent=True, **kwargs):
     def copy_files(self):
         if self.script:
             shutil.copy(self.script, self.script_dst)
-            os.chmod(self.script_dst, 0777)
+            os.chmod(self.script_dst, 0o777)
```

Comment:
The change is okay. However the explanation is misleading. It states:

> For Python 3 permission mode must have a prefix with 0o [1].
>
> 1. https://docs.python.org/3/library/os.html

First, it just the octal number syntax that is accepted on both Python 2.6+ and Python 3. It is not a special permission object or kinda.

Second, the link does not explain anything regarding deprecation of the `0777` octal number syntax.

Reply:
Updated commit message:
```
    python3: fix mode format in chmod() call
    
    flake8 warn about SyntaxError E999: "leading zeros in decimal integer
    literals are not permitted; use an 0o prefix for octal integers".
    Patch fixes an error.
```